app/models/wpcc/your_details_form.rb

```diff
@@ -7,12 +7,26 @@ class YourDetailsForm
 
     GENDERS = %w[male female].freeze
     SALARY_FREQUENCIES = %w[year month fourweeks week].freeze
+    SALARY_THRESHOLDS  = [5876.00, 489.67, 452.00, 113.00].freeze
     CONTRIBUTIONS = %w[full minimum].freeze
+    LOW_SALARY_THRESHOLDS = SALARY_FREQUENCIES.zip(SALARY_THRESHOLDS).to_h
 
     validates :age, presence: true
     validates :gender, inclusion: { in: GENDERS }
     validates :salary, numericality: { only_integer: true, greater_than: 0 }
     validates :salary_frequency, inclusion: { in: SALARY_FREQUENCIES }
     validates :contribution_preference, inclusion: { in: CONTRIBUTIONS }
+
+    # Custom validation
+    validate :low_salary_threshold
```

Comment:
Can we use a validator for this? http://api.rubyonrails.org/classes/ActiveModel/Validator.html We can pair on this if you like.